types/dygraphs/index.d.ts

```diff
@@ -1347,13 +1609,32 @@ declare class Dygraph {
     static DASHED_LINE: number[];
     static DOT_DASH_LINE: number[];
 
+    /**
+     * Standard plotters.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#.Plotters}
+     */
     static Plotters: {
-        errorPlotter: any;
-        linePlotter: any;
-        fillPlotter: any;
-    }
-}
+        errorPlotter(event: any): void;
+        linePlotter(event: any): void;
+        fillPlotter(event: any): void;
+    };
 
-declare module "dygraphs" {
-    export default Dygraph;
+    /**
+     * Point structure.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#.PointType}
+     */
+    static PointType: {
```

Comment:
I found this in the documentation. Should I remove it?

Reply:
If you go to <https://dygraphs.com> and open the console, you can see that it exists but it's `undefined`:

```
> Dygraph.PointType
undefined
```

It's only there because [Google's Closure Compiler][1] used it for type annotations. Users shouldn't use it as a value, so it should be removed.

[1]: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler